packages-exp/auth-compat-exp/src/recaptcha_verifier.ts

```diff
@@ -15,15 +15,17 @@
  * limitations under the License.
  */
 
-import firebase from '@firebase/app';
+import firebase from '@firebase/app-compat';
 import { FirebaseApp } from '@firebase/app-types';
 import * as impl from '@firebase/auth-exp/internal';
 import * as compat from '@firebase/auth-types';
 import * as externs from '@firebase/auth-types-exp';
+import { unwrap, Wrapper } from './wrap';
 
 export class RecaptchaVerifier
```

Comment:
Is it any different than the `RecaptchaVerifier` from `auth-exp`. Can we just reexport the one from `auth-exp`?

Reply:
The auth-exp one needs an unwrapped auth object, which is why this has to be this way